### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by metavaults**

### CODE REVIEW

**Review by angelinrashmi2000 — APPROVED** [review]

File: scrapeIDArtists/server.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+});
+
+server.listen(port, hostname, () => {
+  console.log(`Server running at http://${hostname}:${port}/`);
+
+  var obj = {
+    table: []
+  };
+
+  // obj.table.push({id: 1, type:"person", name:"Lukas Music Star", country: "LT"});
```
Comment (sacdallago) [inline scrapeIDArtists/server.js]: Why is there commented code in this PR? What's the purpose of this code, why is it not uncommented, is it needed?

Comment (sacdallago) [reply scrapeIDArtists/server.js]: If the sense is to have an example of what the data looks like, use the long comments: `\** ....` and describe what you are doing

**Review by sacdallago — DISMISSED** [review]

File: scrapeIDArtists/server.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+var fs = require('fs');
+var async = require('async');
+
+const hostname = '127.0.0.1';
+const port = 3000;
+
+const server = http.createServer((req, res) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'text/plain');
+  res.end('Hello World\n');
```
Comment (kordianbruck) [inline scrapeIDArtists/server.js]: You can run a .js file with node without having to start a http server. If you are only scraping data why do you need a webserver?

File: scrapeIDArtists/server.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+            var url = "http://musicbrainz.org/ws/2/artist/?query=begin:"+date+"&fmt=json";
+
+            console.log(url);
+
+            console.log('request started!');
+            
+            request({
+                url: url,
+                headers: {
+                'User-Agent': 'ClassicalMusic/1.5.0 ( laucius@gmail.com )'
```
Comment (kordianbruck) [inline scrapeIDArtists/server.js]: Be careful with commiting your email - spamers are crawling Github to find those

**Comment by TimHenkelmann** [comment]
Applied tutor suggestions for improvement:
 - removed artists.json output and package.json
- added .gitignore file and artists_example.json
- updated server.js according to tutor input:
    - removed server
    - removed unnecessary comments
    - removed email from request

Ready for review!

**Review by angelinrashmi2000 — APPROVED** [review]

**Comment by TimHenkelmann** [comment]
Hey @sacdallago we implemented your improvement proposals (removing unnecessary comments etc.) but it still says that that changes are requested. Could you help us with that?

**Comment by kordianbruck** [comment]
@TimHenkelmann he probably has not yet checked his emails. I've dismissed his review. You are ready to merge.

**Comment by sacdallago** [comment]
Thanks @kordianbruck :D sorry @TimHenkelmann That's precisely why we tutors should not be handling PRs :) But we do it here just to give you a hand, don't hate us ❤️

**Pull request merged**